### ISSUE TITLE
`newlines_after_classes` false positive

### ISSUE THREAD

**Issue by MoLLIa** [clutchski/coffeelint#535]
Below code (with coffee setters) does not pass validation:
```coffeescript
class User

  setName: (@name) ->

  setAge: (@age) ->


SomeFunction()
```
Settings in `coffeelint.json`:
```javascript
  "newlines_after_classes": {
    "value": 2,
    "level": "error"
  }
```
Error:
```
✗ #4: Wrong count of newlines between a class and other code. Expected 2 got 3.
```

**Comment by swang** [comment]
Thanks for submitting an issue.  This problem is already known about and addressed in #518.
